syntax = "proto3";

package juturna.proto.messaging;

import "payloads.proto";

service MessagingService {
  // Send a message and get acknowledgment
  rpc SendMessage(juturna.proto.payloads.MessageProtoEnvelope) returns (juturna.proto.payloads.MessageProtoEnvelope);
}